packages/components/Callout/src/Callout.types.ts

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import * as ReactNative from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { IFocusable } from '@fluentui-react-native/interactive-hooks';
+
+export const calloutName = 'RNFCallout';
+
+/**
+ * Properties and Tokens for fabric native Callout
+ */
+
+export type DirectionalHint =
+  | 'leftTopEdge'
+  | 'leftCenter'
+  | 'leftBottomEdge'
+  | 'topLeftEdge'
+  | 'topAutoEdge'
+  | 'topCenter'
+  | 'topRightEdge'
+  | 'rightTopEdge'
+  | 'rightCenter'
+  | 'rightBottomEdge'
+  | 'bottonLeftEdge'
+  | 'bottomAutoEdge'
+  | 'bottomCenter'
+  | 'bottomRightEdge';
+
+export interface ICalloutTokens extends IBackgroundColorTokens, IBorderTokens {
```

Comment:
Callout doesn't have UI, why do you extend from IBackgroundColorTokens and IBorderTokens?

Reply:
The beak needs a concept of background, border, even if we push it to child views to mind the main content colors